js/flightlog_fields_presenter.js

```diff
@@ -999,10 +1027,14 @@ function FlightLogFieldPresenter() {
                     }
                 case 'RTH':
                     switch(fieldName) {
+                        case 'debug[0]':
+                            return value.toFixed(0) + 'us'; // rescue throttle 1000-2000
                         case 'debug[1]':
-                            return (value / 100).toFixed(1) + 'deg';
+                            return (value / 100).toFixed(1) + 'deg'; // rescue pitch angle
+                        case 'debug[2]':
+                            return value.toFixed(0) + 'us'; // altitude adjustment min to max throttle
                         default:
-                            return value.toFixed(0);
+                            return value.toFixed(0); // coded values to 2000
```

Comment:
I think this is the only one missing. The debug[0] and debug[2] are the same, so group this two and I think we are ready.

Reply:
Thank you.  I hope it is fixed now.
Should I squash?